src/locales/cy/pages.yml

```diff
@@ -1,10 +1,10 @@
 intro:
   h1: Intro
 rti-payslip-national-insurance:
-  title: Enter the amount of National Insurance you paid from a recent payslip
-  inset: Use a payslip dated after {{ threeMonthsAgo }} to answer this question
+  title: Rhowch swm yr Yswiriant Gwladol a dalwyd gennych o slip cyflog diweddar
+  inset: Defnyddiwch slip cyflog wedi'i ddyddio ar ôl {{ threeMonthsAgo }} i ateb y cwestiwn hwn
```

Comment:
We'll need to make sure the date (month) gets translated too - we should handle that in [PYIC-5211](https://govukverify.atlassian.net/browse/PYIC-5211)